docs/help_out.txt

```diff
@@ -33,6 +37,9 @@ GLOBAL OPTIONS:
    --eigenda.disable-tls                                                    Disable TLS for gRPC communication with the EigenDA disperser. Default is false. (default: false) [$EIGENDA_PROXY_EIGENDA_GRPC_DISABLE_TLS]
    --eigenda.disperser-rpc value                                            RPC endpoint of the EigenDA disperser. [$EIGENDA_PROXY_EIGENDA_DISPERSER_RPC]
    --eigenda.eth-rpc value                                                  URL of the Ethereum RPC endpoint. Needed to confirm blobs landed onchain. [$EIGENDA_PROXY_EIGENDA_ETH_RPC]
+   --eigenda.max-blob-length value                                          Maximum blob length to be written or read from EigenDA. Determines the number of SRS points
```

Comment:
just to be sure, this is just being moved from the KZG category (got mislabeled in previous V2 PR I believe?) to here right? But flag itself isnt changing?

Reply:
> just to be sure, this is just being moved from the KZG category

Correct.

> (got mislabeled in previous V2 PR I believe?)

I don't think so- it was just in the verification category all along, and was being propagated through the rest of the config structs in a roundabout way.